api/api.go

```diff
@@ -13,7 +13,8 @@ type MetalConfig struct {
 	NTPServers  []string `json:"ntp_servers,omitempty"`
 	Partition   string   `json:"partition"`
 	// Logging contains logging configurations passed to metal-hammer
-	Logging *Logging `json:"logging,omitempty"`
+	Logging   *Logging     `json:"logging,omitempty"`
+	OciConfig []*OciConfig `json:"oci_config,omitempty"`
```

Comment:
You need to wire up the configuration of the ociconfig in the cli:
https://github.com/metal-stack/pixie/blob/main/pixiecore/cli/grpccmd.go